libs/domains/services/data-access/src/lib/domains-services-data-access.ts

```diff
@@ -171,6 +171,13 @@ export const services = createQueryKeys('services', {
           serviceType: ServiceTypeEnum.JOB as const,
         }))
         .exhaustive()
+      if (service.memory) {
+        service.memory *= 10 ** 6 // API return memory in MB
+      }
+      if (service.serviceType === 'DATABASE' && service.storage) {
+        service.storage *= 10 ** 9 // API return memory in GB
+      }
```

Comment:
@RemiBonnet  I think it could be a good thing to only use Byte internally.
Ready to discuss it if you disagree.

Reply:
Yep, not sure understand, available when you want